build.gradle

```diff
@@ -1,19 +1,9 @@
-buildscript {
-    repositories {
-        mavenCentral()
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.github.jengelman.gradle.plugins:shadow:4.0.2'
-        classpath 'net.saliman:gradle-cobertura-plugin:2.5.4'
-        classpath 'com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3'
```

Comment:
I won't mourn the loss of the Bintray plugin, but was it deliberate? (The latest version is 1.8.5 anyway.)

Reply:
Yes deliberate, cannot find any bintray closures in build.gradle , so removed it.